src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -322,7 +322,7 @@ typedef struct uct_ib_mlx5_iface_config {
 #endif
     uct_ib_mlx5_mmio_mode_t  mmio_mode;
     ucs_ternary_auto_value_t ar_enable;
-    int                      cqe_zipping_enable;
+    unsigned                 cqe_zipping_dirs_bitmap;
```

Comment:
Minor: usually adding the var type to its name is redundant and "bad practice".
e.g we don't usually name things `..._arr`, `..._struct` etc.
Let's remove the `_bitmap`.